src/WebServer_DevicesPage.ino

```diff
@@ -704,6 +718,32 @@ void handle_devices_TaskSettingsPage(taskIndex_t taskIndex, byte page)
       }
     }
 
+#ifdef FEATURE_I2CMULTIPLEXER
+    // Show selector for an I2C multiplexer port if a multiplexer is configured
+    if (Device[DeviceIndex].Type == DEVICE_TYPE_I2C && Settings.I2C_Multiplexer_Addr != -1) {
+
+      addFormSubHeader(F("I2C Multiplexer"));
+
+      int taskDeviceI2CMuxPort = Settings.I2C_Multiplexer_Port[taskIndex];
+      String  i2c_mux_portoptions[9];
```

Comment:
In general, when using these stack allocated arrays (N.B. also is applicable for heap allocated), please wrap their definition and usage in a small scope using 
```c++
{
  String bla[9];
   ...
   function(bla);
}
```
This way, the data is removed from the stack as soon as you try to call the other functions.
And since the `String` object is both stack and heap allocated, you may just prevent a crash due to memory issues when these Strings may have become large or fragment the heap.

Reply:
Ah, yes, I try to do that every time, but this one I forgot, it seems.